src/sentry/utils/suspect_resolutions/metric_correlation.py

```diff
@@ -14,7 +14,7 @@ def is_issue_error_rate_correlated(
     resolution_time = resolved_issue.resolved_at
 
     start_time = resolution_time - timedelta(hours=5)
-    end_time = resolution_time + timedelta(hours=5)
+    end_time = resolution_time + timedelta(hours=1)
```

Comment:
The calculation of error-rate correlation could be made smarter by looking for a significant dip in the number of events at some point instead of relying on the resolution time (the time that the issue was _marked_ as resolved may not always align with when the issue was actually resolved). For now, since we're using the resolution time, we can set the end time of the calculation to be an hour after the resolution time - if both issues are resolved, the number of events should be consistent (and close to 0) for both issues.